examples/all-clusters-app/ameba/main/include/ManualOperationalStateCommand.h

```diff
@@ -0,0 +1,213 @@
+ *    limitations under the License.
+ */
+
+#include "controller/InvokeInteraction.h"
+#include "controller/ReadInteraction.h"
+#include <OperationalStateManager.h>
+
+#if CONFIG_ENABLE_CHIP_SHELL
+#include "lib/shell/Engine.h"
+#include "lib/shell/commands/Help.h"
+#endif // ENABLE_CHIP_SHELL
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::OperationalState;
+
+#if CONFIG_ENABLE_CHIP_SHELL
+using Shell::Engine;
+using Shell::shell_command_t;
+using Shell::streamer_get;
+using Shell::streamer_printf;
+
+Engine sShellManualOperationalStateSubCommands;
+Engine sShellManualRVCOperationalStateSubCommands;
+#endif // defined(ENABLE_CHIP_SHELL)
+
+#if CONFIG_ENABLE_CHIP_SHELL
+/********************************************************
+ * Operational State Functions
+ *********************************************************/
+
+CHIP_ERROR ManualOperationalStateCommandHelpHandler(int argc, char ** argv)
+{
+    sShellManualOperationalStateSubCommands.ForEachCommand(Shell::PrintCommandHelp, nullptr);
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ManualOperationalStateCommandHandler(int argc, char ** argv)
+{
+    if (argc == 0)
+    {
+        return ManualOperationalStateCommandHelpHandler(argc, argv);
+    }
+
+    return sShellManualOperationalStateSubCommands.ExecCommand(argc, argv);
+}
+
+CHIP_ERROR ManualOperationalStateSetStateCommandHandler(int argc, char ** argv)
+{
+    if (argc != 1)
+    {
+        return ManualOperationalStateCommandHelpHandler(argc, argv);
+    }
+    uint32_t state = atoi(argv[0]);
+
+    CHIP_ERROR err;
+    err = GetOperationalStateInstance()->SetOperationalState(state);
+
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(DeviceLayer, "ManualOperationalStateSetStateCommandHandler Failed!\r\n");
+    }
+
+    return err;
+}
+
+CHIP_ERROR ManualOperationalStateSetErrorCommandHandler(int argc, char ** argv)
+{
+    if (argc != 1)
+    {
+        return ManualOperationalStateCommandHelpHandler(argc, argv);
+    }
+
+    GenericOperationalError err(to_underlying(ErrorStateEnum::kNoError));
+    uint32_t error = atoi(argv[0]);
+
+    switch (error)
+    {
+    case to_underlying(OperationalState::ErrorStateEnum::kNoError):
+        err.errorStateID = to_underlying(OperationalState::ErrorStateEnum::kNoError);
+        break;
+    case to_underlying(OperationalState::ErrorStateEnum::kUnableToStartOrResume):
+        err.errorStateID = to_underlying(OperationalState::ErrorStateEnum::kUnableToStartOrResume);
+        break;
+    case to_underlying(OperationalState::ErrorStateEnum::kUnableToCompleteOperation):
+        err.errorStateID = to_underlying(OperationalState::ErrorStateEnum::kUnableToCompleteOperation);
+        break;
+    case to_underlying(OperationalState::ErrorStateEnum::kCommandInvalidInState):
+        err.errorStateID = to_underlying(OperationalState::ErrorStateEnum::kCommandInvalidInState);
+        break;
+    default:
```

Comment:
This might be a lot more readable as:
```suggestion
    case to_underlying(OperationalState::ErrorStateEnum::kNoError):
    case to_underlying(OperationalState::ErrorStateEnum::kUnableToStartOrResume):
    case to_underlying(OperationalState::ErrorStateEnum::kUnableToCompleteOperation):
    case to_underlying(OperationalState::ErrorStateEnum::kCommandInvalidInState):
        err.errorStateID = error;
        break;
    default:
```

Reply:
Updated to more readable switch case